djangoprojects/provision.py

```diff
@@ -0,0 +1,97 @@
+"""Greeter.
+
+    Usage:
+      provision.py create <name>
+      provision.py list <name>
+      provision.py delete <name>
+      provision.py (-h | --help)
+
+    Options:
+      -h --help     Show this screen.
+
+"""
+import json
+import time
+import requests
+import environ
+from docopt import docopt
+
+env = environ.Env(
+    # set casting, default value
+    DEBUG=(bool, True)
+)
+
+# reading .env file
+environ.Env.read_env()
+
+api_token = env('API_KEY')
+api_url_base = 'https://api.digitalocean.com/v2/'
+headers = {'Content-Type': 'application/json', 'Authorization': 'Bearer {0}'.format(api_token)}
+
+
+def list(droplet_name):
+    api_droplet = api_url_base + 'droplets?tag_name=' + droplet_name
+    response = requests.get(api_droplet, headers=headers)
+    jsondata = json.loads(response.text)
+    try:
+        if droplet_name == "Name:" + (jsondata.get("droplets")[0]).get("name") and response.status_code == 200 :
+            print(response.content.decode('utf-8'))
+        else:
+            print("FAILED TO GET RESPONSE")
+    except IndexError:
+        print("DROPLET DOESNT EXIST")
+
+
+def create(droplet_name):
+    data = {
+             "name": droplet_name,
+             "region": "nyc3",
+             "size": "s-1vcpu-1gb",
+             "image": "ubuntu-16-04-x64",
+             "ssh_keys": [23341984],
```

Comment:
Dont hardcode your key here. Get your key from the file, compute the fingerprint and use that in `ssh_keys` params